src/main/java/com/ibs/kpitrackerservice/controller/KpiController.java

```diff
@@ -1,7 +1,44 @@
 package com.ibs.kpitrackerservice.controller;
 
-import org.springframework.web.bind.annotation.RestController;
+import com.ibs.kpitrackerservice.model.Kpi;
+import com.ibs.kpitrackerservice.service.KpiService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
+@RequestMapping(path = "/kpi")
 public class KpiController {
+    @Autowired
+    private KpiService kpiService;
+
+    @GetMapping(path = "/{id}")
+    public ResponseEntity<Kpi> getKpi(@PathVariable("id") String id) {
+        return ResponseEntity.ok(kpiService.getKpi(id));
+    }
+
+    @GetMapping(path = "/getAll")
```

Comment:
we would need to follow the same rest api naming conventions throughout the application. Please check the TeamController/UserController and correct it accordingly.